content/fast/adaptive-serving-based-on-network-quality/index.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: guide
+title: Adaptive serving based on network quality
+author: mihajlija
+description: |
+  Using Network Information API to adapt which assets you're serving to users based on the quality of their connection.
+web_lighthouse: N/A
+web_updated_on: 2019-04-01
+web_published_on: 2019-04-01
+wf_blink_components: N/A
+---
+
+# Adaptive serving based on network quality
+
+Loading a website can be a very different experience depending on the network conditions. Everything is usually smooth when you are on a fast network, but when you are on the go with a humble data plan and spotty connection, or stuck with a laptop on a sluggish coffee shop wifi, it's a different story.
```

Comment:
Small suggestion to change second you are to 'you're': 'but when you're on the go'.